### PR TITLE
Add Offline OPD Visit Count Report

### PR DESCRIPTION
**Pull request by poojadeshpande01**
Add Offline OPD Visit Count Report

### CODE REVIEW

**Comment by poojadeshpande01** [comment]
image

**Comment by mohan-13** [comment]
Integration user should not be shown. Those are patients synced from online to offline

**Review by riyaTw — APPROVED** [review]

**Review by riyaTw — CHANGES_REQUESTED** [review]
Can we change Column name as "Location"

**Review by riyaTw — APPROVED** [review]

File: openmrs/apps/reports/sql/offlineOPDVisitCount.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+SELECT u.name AS User, c.name AS Location, COUNT(DISTINCT e.individual_id) AS "Offline OPD Visit Count"
```
Comment (mohan-13) [inline openmrs/apps/reports/sql/offlineOPDVisitCount.sql:1]: Count of distinct individual id will give only the number of patients that had an encounter in the date range. This should be grouped by date also and then sum should be taken

**Review by mohan-13 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**